src/deluge/dsp/delay/delay.cpp

```diff
@@ -32,17 +29,8 @@ Delay::Delay() {
 	repeatsUntilAbandon = 0;
 	prevFeedback = 0;
 
-	// I'm so sorry, this is incredibly ugly, but in order to decide the default sync level, we have to look at the current song, or even better the one being preloaded.
-	Song* song = preLoadedSong;
-	if (!song) {
-		song = currentSong;
-	}
-	if (song) {
-		syncLevel = (SyncLevel)(8 - (song->insideWorldTickMagnitude + song->insideWorldTickMagnitudeOffsetFromBPM));
-	}
-	else {
-		syncLevel = (SyncLevel)(8 - FlashStorage::defaultMagnitude);
-	}
+	syncLevel = (SyncLevel)5;
```

Comment:
Does this get overwritten later?

Reply:
Yeah it's set by mod controllable audio/global effectable now in order to strip out all the dependencies on song and param